src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordPrintMembers.cs

```diff
@@ -119,7 +119,11 @@ internal override void GenerateMethodBody(TypeCompilationState compilationState,
             var F = new SyntheticBoundNodeFactory(this, ContainingType.GetNonNullSyntaxNode(), compilationState, diagnostics);
             try
             {
-                ImmutableArray<Symbol> printableMembers = ContainingType.GetMembers().WhereAsArray(m => isPrintable(m));
+                ArrayBuilder<Symbol> printableMembersBuilder = ArrayBuilder<Symbol>.GetInstance();
+                printableMembersBuilder.AddRange(ContainingType.GetFieldsToEmit());
```

Comment:
:memo: other synthesized record methods use `GetFieldsToEmit` (which already filters tuple error field symbols). But for print members, we want both fields and properties